rclcpp/src/rclcpp/parameter_map.cpp

```diff
@@ -81,7 +81,15 @@ rclcpp::parameter_map_from(const rcl_params_t * const c_params, const char * nod
         throw InvalidParametersException(message);
       }
       const rcl_variant_t * const c_param_value = &(c_params_node->parameter_values[p]);
-      params_node.emplace_back(c_param_name, parameter_value_from(c_param_value));
+      rclcpp::ParameterValue value;
+      try {
+        value = parameter_value_from(param);
```

Comment:
where is this `param` from?

```suggestion
        value = parameter_value_from(c_param_value);
```